replication/README.md

```diff
@@ -366,6 +367,23 @@ message GetVolumeReplicationInfoResponse {
   // Holds the last sync time.
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
+  // Holds the last sync duration.
+  // A Duration represents a signed, fixed-length span of time represented
+  // as a count of seconds and fractions of seconds at nanosecond
+  // resolution. It is independent of any calendar and concepts like "day"
+  // or "month". It is related to Timestamp in that the difference between
+  // two Timestamp values is a Duration and it can be added or subtracted
+  // from a Timestamp. Range is approximately +-10,000 years.
```

Comment:
This documentation is already available at https://developers.google.com/digital-asset-links/reference/rpc/google.protobuf#:~:text=Duration%20(message)-,Duration,of%20seconds%20at%20nanosecond%20resolution. This can be removed